docs/user/debugging/ios.md

```diff
@@ -2,6 +2,14 @@
 
 For debugging and validation purposes on iOS, Glean makes use of a custom URL scheme which is implemented _within the application_ that is consuming Glean.  Glean provides some convenience functions to facilitate this, but it's up to the consuming application to enable this functionality.  Applications that enable this Glean SDK feature will be able to launch the application from a URL with the Glean debug commands embedded in the URL itself.
 
+Debugging features in iOS can also be enabled using environment variables.
+For more information on the available features accessible through this method
+and how to enable them, see [Enabling debugging features through environment variables](./index.md).
+
+These environment variables must be set on the device that is running the application.
+
+> **Note** To set environment variables to the process running your app in an iOS device or emulator you need to edit the scheme for your app. In the Xcode IDE, you can use the shortcut `Cmd + <` to open the scheme editor popup. The environment variables editor is under the `Arguments` tab on this popup.
```

Comment:
Hey @travis79 , do you know any other ways to do this and do you think this description is clear?

Reply:
One thing I would ask is can we put this under a separate heading?  I think it may lead the user to believe that they must set the environment variables to use the other iOS debugging methods, so anything we can do to indicate that these are separate and exclusive debugging methods.